docs/jitserver_tuning.md

```diff
@@ -44,9 +44,12 @@ The client-session caches are deleted when the clients terminate, but this can h
 
  A JITServer instance can have several AOT caches, each with its own name. This addresses the situation when client JVMs with significantly different profiles of execution use the same JITServer instance. A client JVM can indicate a specific AOT cache it wants to use by providing its name with the following command line option [`-XX:JITServerAOTCacheName=<cache_name>`](xxjitserveraotcachename.md). The default is to use a nameless cache.
```

Comment:
The sentence "The default is to use a nameless cache" is no longer true, because we assigned the name "default" to the implicit/nameless cache. Therefore it could be rephrased to:
"If the client doesn't specify a name for the AOT cache, the server will use a cache called "default".